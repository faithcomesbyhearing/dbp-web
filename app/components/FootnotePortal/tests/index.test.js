import React from 'react';
import Enzyme from 'enzyme';
import Adapter from '@cfaester/enzyme-adapter-react-18';
import FootnotePortal from '..';

Enzyme.configure({ adapter: new Adapter() });

// Mocks
Object.defineProperty(document, 'getElementById', {
	value: (input) => {
		const el = document.createElement('div');
		el.id = input;
		return el;
	},
});

const props = {
	message: 'Please Login.',
	coords: { x: 15, y: 15 },
	closeFootnote: jest.fn(),
};

describe('FootnotePortal component', () => {
	it('should match snapshot with expected props', () => {
		const tree = Enzyme.mount(<FootnotePortal {...props} />);
		expect(tree).toMatchSnapshot();
	});
});
