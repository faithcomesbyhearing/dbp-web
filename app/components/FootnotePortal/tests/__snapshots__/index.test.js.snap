// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FootnotePortal component should match snapshot with expected props 1`] = `ReactWrapper {}`;
